### PR TITLE
i18n(ko-KR): translate `sidebar.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

translate `sidebar.mdx`

### CODE REVIEW

File: docs/src/content/docs/ko/guides/sidebar.mdx [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+<SidebarPreview
+	config={[
+		{
+			label: '가이드',
+			items: [
+				{ label: '홈페이지', link: '#' },
+				{
+					label: '사용자 정의 사이드바 라벨',
+					link: '#',
+					badge: { text: 'New', variant: 'tip' },
```
Comment (delucis) [inline docs/src/content/docs/ko/guides/sidebar.mdx:225]: Same here, that “New” can be translated:
````suggestion
    text: 새로운
    variant: tip
---
```

위 frontmatter가 있는 페이지를 포함하는 자동 생성 그룹은 다음 사이드바를 생성합니다.

<SidebarPreview
	config={[
		{
			label: '가이드',
			items: [
				{ label: '홈페이지', link: '#' },
				{
					label: '사용자 정의 사이드바 라벨',
					link: '#',
					badge: { text: '새로운', variant: 'tip' },
````

**Review by delucis — COMMENTED** [review]
This looks great @jsparkdev! I noticed a few places where the “New” badge isn’t translated. That’s OK if it’s intentional, but I’d definitely translate the first on in the frontmatter so it shows up in the Starlight navigation in Korean.

**Comment by jsparkdev** [comment]
The badge “New” is also used very frequently on various websites in South Korea.

So, I intentionally didn't translate the "New" badge.

If many other Koreans reading this document feel uncomfortable, I will think about revising it.

Thank you for your good comments and thorough review! @delucis 😊

File: docs/src/content/docs/ko/guides/sidebar.mdx [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+
+<SidebarPreview
+	config={[
+		{
+			label: '가이드',
+			items: [
+				{
+					label: '컴포넌트',
+					link: '/ko/guides/components/',
+					badge: { text: 'New', variant: 'default' },
```
[the remark this answers is not in the document]
Comment (jsparkdev) [reply docs/src/content/docs/ko/guides/sidebar.mdx:271]: Can you merge this PR? 🤣

**Review by delucis — APPROVED** [review]
OK, great — that sounds good to me then! Thanks again for the translation and your patience waiting to get this merged 🙌

**Pull request merged**